Source/Scene/Expression.js

```diff
@@ -1060,8 +1104,17 @@ define([
             return Cartesian3.add(left, right, ScratchStorage.getCartesian3());
         } else if ((right instanceof Cartesian4) && (left instanceof Cartesian4)) {
             return Cartesian4.add(left, right, ScratchStorage.getCartesian4());
+        } else if ((typeof(left) === 'string') || (typeof(right) === 'string')) {
+            return left + right;
```

Comment:
See what JavaScript says about `toString` functions.  I want to say yes, we should state how, so things are very well specified, but we would also have to think about how many decimal places, etc.